src/sql/workbench/contrib/notebook/browser/notebookActions.ts

```diff
@@ -327,6 +338,14 @@ export class RunParametersAction extends TooltipFromLabelAction {
 				for (let parameter of cell.source) {
 					// Only add parameters that contain the proper parameters format (ex. x = 1) shown in the Parameterization Doc.
 					if (parameter.includes('=')) {
+						// Multi-line parameters are currently not supported
+						if (parameter.includes(';')) {
```

Comment:
So now any parameters that include a ; in them will not work, regardless of whether they're actually valid or not (so a string parameter with a ; in it means they just will never be able to run it). That doesn't seem like it's worth the small gain  here.